.github/workflows/first_interaction.yml

```diff
@@ -0,0 +1,52 @@
+name: 🔗 GHA
+
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    branches: [master]
+    types: [opened]
+
+jobs:
+  check_for_first_issue:
+    name: 🌱 Check for first interaction
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: |
+                         Hello! 👋
+
+                         First of all, thanks for taking the time to report an issue here.
+
+                         If not done already, be sure to join a MRP (minimal reproducible
```

Comment:
I am totally missing something here. Why saying "join"? Is it because GitHub uses that term? "Append", "Attach" a file is usually more common.

Reply:
It's my French, sorry! I'll replace it with append.